holoviews/tests/ui/bokeh/test_callback.py

```diff
@@ -414,6 +418,7 @@ async def popup_form(x, y):
         locator = page.locator(".bk-btn")
         expect(locator).to_have_count(2)
 
+    @pytest.mark.filterwarnings("ignore:reference already known")
```

Comment:
I tried to see if I could figure out where this was coming from but without luck. It is the button ID that is flagged.

``` python
import numpy as np
import panel as pn

import holoviews as hv

hv.extension("bokeh")

rng = np.random.default_rng(10)
points =  hv.Points(rng.normal(size=(1000, 2)))

def popup_form(x, y):
    def hide(_):
        button.visible = False

    button = pn.widgets.Button(name=f"{x},{y}", on_click=hide)
    return button

points.opts(tools=["tap"])
hv.streams.Tap(source=points, popup=popup_form)

pn.panel(points).servable()
```

Reply:
@philippjfr, you are welcome to give it a go.